templates/workspace_services/guacamole/guacamole-server/guacamole-auth-azure/src/main/java/org/apache/guacamole/auth/azuretre/connection/ConnectionService.java

```diff
@@ -71,9 +71,10 @@ private static Map<String, GuacamoleConfiguration> getConfigurations(final Azure
                     if (templateParameters.has("hostname") && templateParameters.has("ip")) {
                         final String azureResourceId = templateParameters.getString("hostname");
                         final String ip = templateParameters.getString("ip");
-                        setConfig(config, azureResourceId, ip);
+                        final String display_name = templateParameters.getString("display_name");
+                        setConfig(config, azureResourceId, ip, display_name);
                         LOGGER.info("Adding a VM: {}", ip);
```

Comment:
you can also add the display name to the log line

Reply:
Done